projects/configs/tag-operations/mri.yaml

```diff
@@ -386,3 +386,39 @@
   group: 0x7fe0
   element: 0x0010
   op: "keep"
+- name: "Position Reference Indicator"
+  group: 0x0020
+  element: 0x1040
+  op: "replace"
+- name: "Patient Position"
+  group: 0x0018
+  element: 0x5100
+  op: "replace"
+- name: "Study Date"
+  group: 0x0008
+  element: 0x0020
+  op: "replace"
+- name: "General Study"
+  group: 0x0008
+  element: 0x0030
+  op: "replace"
+- name: "MR Image"
+  group: 0x0018
+  element: 0x0022
+  op: "replace"
+- name: "Patient"
+  group: 0x0010
+  element: 0x0030
+  op: "replace"
+- name: "Referring Physician's Name"
+  group: 0x0008
+  element: 0x0090
+  op: "replace"
+- name: "Bits Allocated"
+  group: 0x0028
+  element: 0x0100
+  op: "keep"
+- name: "Patient's Sex"
+  group: 0x0010
+  element: 0x0040
+  op: "keep"
```

Comment:
we should add these in order of group and then element, like we've done in the rest of the file

Reply:
just ordered yaml here [3001682](https://github.com/SAFEHR-data/PIXL/pull/493/commits/30016828dc759b3e05f140a9b6df9b0af046b53e)